cli/plasmo/templates/static/react17/content-script-ui-mount.tsx

```diff
@@ -1,28 +1,41 @@
-// @ts-nocheck
-// prettier-sort-ignore
 import React from "react"
 
+// prettier-sort-ignore
+// @ts-ignore
 import * as RawMount from "__plasmo_mount_content_script__"
+
 import * as ReactDOM from "react-dom"
 
+import type { PlasmoCSUI } from "../../../src/type"
+
 // Escape parcel's static analyzer
-const Mount = RawMount
+const Mount = RawMount as PlasmoCSUI
+
+const mountState = {
+  document: document || window.document,
+  observer: null as MutationObserver | null,
+  shadowHost: null as Element | null,
+
```

Comment:
remove space

Reply:
The top group and the bottom group have certain context in my mind. I'm not sure what comments I would put there yet that's why there's a white space but technically there should be a comment.